tests/testthat/test-req-url.R

```diff
@@ -15,8 +15,6 @@ test_that("automatically adds /", {
   expect_equal(req_url_path_append(req1, "index.html")$url, "http://example.com/index.html")
   expect_equal(req_url_path_append(req1, "/index.html")$url, "http://example.com/index.html")
   expect_equal(req_url_path_append(req2, "index.html")$url, "http://example.com/index.html")
-  # specifically requested to add in both
-  expect_equal(req_url_path_append(req2, "/index.html")$url, "http://example.com//index.html")
```

Comment:
Can you keep this test but update the expected value?

Reply:
Done, [14b7aa0](https://github.com/r-lib/httr2/pull/136/commits/14b7aa00ea0ddb1db33b83c85faf0c3df7d1c72c) has that in it.